modules/local/clincnv/bedannotategc/main.nf

```diff
@@ -0,0 +1,44 @@
+process BEDANNOTATEGC {
```

Comment:
```suggestion
process CLINCNV_BEDANNOTATEGC {
```

I think adding the `CLINCNV_` prefix to all these modules will help in keeping it clear that these modules are part of the clincnv toolkit. Can this be added to each created process here?

Reply:
Or NVM this is part of `ngs-bits`. Can this be updated to the `NGSBITS_` prefix instead and be moved to a correct subfolder called `ngsbits`? This will keep it clear from where the tool comes :)